### ISSUE TITLE
and the symbol (@)

### ISSUE THREAD

**Issue by mama21mama** [redmatrix/redmatrix#335]
<a href="http://upimage.mamalibre.com.ar/#4de37b2a49f377be7c9fe11baa81d84a.png"><img src="http://upimage.mamalibre.com.ar/up/4de37b2a49f377be7c9fe11baa81d84a.png" alt="Image hosting by Imagebin" /></a>

**Comment by redmatrix** [comment]
We are not using the @ symbol but a unicode symbol which looks just like it. This way the addresses cannot be scraped as email addresses and spammed. It seems that your default browser font is missing this unicode character. Unfortunately we don't have any control over installed fonts, and selecting a font specifier that will work cross-platform for some symbols is very hard. If you provide your platform (browser and operating system) I'll see if I can come up with something.

**Comment by mama21mama** [comment]
debian os
seamonkey browser

**Comment by redmatrix** [comment]
I don't have a Debian platform handy but it did display on Ubuntu 14.04 with the latest Seamonkey. I'm assuming that the default font for this page is freesans on Linux (this is what we've specified). I've checked with LibreOffice and the character &xFF20; displays fine in FreeSans, so I would recommend installing or updating that font. It's possible that Debian stable version of FreeSans is a bit older and may be lacking some of these extended characters. I don't have an easy way of checking that currently.
